src/pages/journal/components/empty-slot/empty-slot.scss

```diff
@@ -0,0 +1,47 @@
+empty-slot {
+  ion-card {
+    &.grey-bg {
+      background: transparent;
+    }
+    border: 4px dashed #4A4A4A;
```

Comment:
The design in abstract has the border width set at `2px`

Reply:
Done